### DISCUSSION TITLE
Taiko no Tatsujin compatibility - Why is it not possible/done yet?

### DISCUSSION THREAD

**Discussion by Alidonis** [mamedev/discussions#95] [General]
I'm using a Taiko10 ROM. However, despite this slight hope of compatibility I had, the game did not even boot. I'd like to know what happens under the hood of the Namco System 2x6 and why such games won't boot on mame.

Any replies are really appreciated

**Comment by cuavas** [comment]
MAME does not emulate Sony PlayStation 2 hardware, or the related Namco System 246 and System 256 platforms.  Only the Taiko no Tatsujin games on the Namco System 10 platform (PSone-related) are playable.
